lib/rubocop/rspec/expect_offense.rb

```diff
@@ -77,26 +106,31 @@ def self.parse(annotated_source)
           source      = []
           annotations = []
 
-          annotated_source.each_line do |source_line|
+          source_area, _seperator, correction =
+            annotated_source.partition(CORRECTION_SEPARATOR)
```

Comment:
The nitpick of alle nitpicks: The explicitly ignored variable has a typo 😄 Should be `_separator`.

Actually, I think you can just use `#split` instead of `#partition`, and you won’t even have the ignored variable anymore. Although that would set `correction` to `nil` instead of `""`, which would then need to be properly handled in the initializer.

Reply:
I had initially used `split`, but I wound up switching to partition for some reason that I can't remember. `split` appears to be working so I will go back to using that.